### PR TITLE
Fix "Unable to start pyls" for Debian

### PR DESCRIPTION
**Pull request by simon123h**
This PR fixes my issue https://github.com/lgeiger/ide-python/issues/31 on my Debian machine.
Definitely needs further review as it affects all linux machines!

As with the already implemented workaround for Mac, this is not more than a workaround and might not get down to the root of the problem.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ class PythonLanguageClient extends AutoLanguageClient {
   }
 
   async startServerProcess(projectPath) {
-    // For some reason Atom only detects the correct env if started from the command line on Mac.
-    const env = process.platform === "darwin" ? await shellEnv() : process.env;
+    // For some reason Atom only detects the correct env if started from the command line on Mac and Debian.
+    const env = process.platform === "darwin" || process.platform === "darwin" ? await shellEnv() : process.env;
```
Comment (lgeiger) [inline lib/main.js]: This doesn't change behavior.

Comment (simon123h) [reply lib/main.js]: Ough, my bad! It was supposed to be `linux` not `darwin` in the second condition.

Comment (lgeiger) [reply lib/main.js]: Unfortunately this will re-introduce #20 😕

**Comment by kbrose** [comment]
Maybe there's some combination of these checks and using `await atom.updateProcessEnv();` as in https://github.com/lgeiger/ide-python/pull/28/files to make this work in all cases?

**Pull request closed without merging**